slideruler-port.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/SlideruleActivity.py b/SlideruleActivity.py
+index c167d04..5c9c29d 100644
+--- a/SlideruleActivity.py
++++ b/SlideruleActivity.py
+@@ -31,7 +31,7 @@ import gi
+ gi.require_version('Gtk', '3.0')
+ 
+ from gi.repository import Gtk, Gdk
+-
++from sugarapp.widgets import SugarCompatibleActivity
+ from sugar3.activity import activity
+ from sugar3.graphics.toolbarbox import ToolbarBox
+ from sugar3.bundle.activitybundle import ActivityBundle
+@@ -70,11 +70,12 @@ FUNCTIONS = [FL_add, FC_multiply, FCI_divide, FA_square, FK_cube, FS_sin,
+              FT_tan, FE_natural_log, UK_unknown]
+ 
+ 
+-class SlideruleActivity(activity.Activity):
++class SlideruleActivity(SugarCompatibleActivity):
+     """ A sliderule activity for Sugar """
+ 
+     def __init__(self, handle):
+-        super(SlideruleActivity,self).__init__(handle)
++
```

Comment:
no need for the empty line

Reply:
i'm assuming you meant the empty line above `super().__init__(handle)`, not below